portal-parent/ozwillo-portal-front/pom.xml

```diff
@@ -113,6 +119,12 @@
             <artifactId>memcached-session-manager-tc8</artifactId>
         </dependency>
 
+        <!-- Lombok -->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
Funny (sort of) that this shows up just hours/minutes after I complain on Twitter about how Lombok is unfortunately still a thing in late 2015.
Do what you want (it's not my project), but I'd highly suggest avoiding Lombok (been there, done that)
(oh my, and you only use it to generate a logger static field‽ that's the job of an IDE, not a compile-time hideously hackish thing)